### PR TITLE
Ig reels

### PR DESCRIPTION
**Pull request by tbone1020**
[#1272](https://github.com/wb-apps/tmz-cms/issues/1272)

### CODE REVIEW

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx [hunk 1]
```diff
@@ -103,6 +93,8 @@ export default class InstagramMediaBlockModal extends React.Component {
     let { errorMsg, hideCaption, id, isValid } = this.state;
     const input = event.target.value;
     const mediaId = getInstagramMediaId(input);
+    console.clear();
+    console.log('handleChangeTextarea ~ mediaId', mediaId);
```
Comment (rsumilang) [inline src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx:97]: Remove unneeded console logs.

**Comment by rsumilang** [comment]
@tbone1020 conflicts

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx (hunk 1 above)
Comment (tbone1020) [reply src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx]: @rsumilang logs were removed

**Comment by tbone1020** [comment]
@rsumilang logs were removed and resolved conflicts

**Review by rsumilang — APPROVED** [review]

**Pull request merged**